src/modules/authentication/types/session.d.ts

```diff
@@ -1,6 +1,11 @@
 declare module 'express-session' {
     interface SessionData {
         redirectUrl?: string;
+        passport?: {
+            user: {
+                redirect_uri?: string;
+            };
+        };
```

Comment:
Why was this change made? Are these two different redirect URLs? 
If there is a good reason for this, please add a comment explaining it.
If there isn't a good reason this should be removed.